test/controllers/sessions_controller.js

```diff
@@ -0,0 +1,69 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../../app'),
+  should = chai.should(),
+  User = require('../../app/models').user,
+  userService = require('../../app/services/user_service');
+
+describe('Sessions Controller', () => {
+  describe('/users/session POST', () => {
+    context('when a user exists', () => {
+      beforeEach(done => {
+        userService
+          .createUser({
+            password: '1234567890',
+            email: 'federico.dinucci@wolox.com.ar',
+            firstName: 'Federico',
+            lastName: 'Di Nucci'
+          })
+          .then(() => done());
+      });
+      context('and the information is valid', () => {
+        it('should return created status', done => {
+          chai
+            .request(server)
+            .post('/users/sessions')
+            .send({
+              password: '1234567890',
+              email: 'federico.dinucci@wolox.com.ar'
+            })
+            .then(res => {
+              res.should.have.status(201);
```

Comment:
validate creation with the model

Reply:
Actually its not creating a Model, its creating a Session that is not persisted in any model.